src/main/host/thread_ptrace.c

```diff
@@ -312,55 +330,61 @@ static pid_t _threadptrace_fork_exec(const char* file, char* const argv[],
     switch (pid) {
         case -1: {
             error("fork: %s", g_strerror(errno));
-            return -1;
+            break;
         }
```

Comment:
(Optional) I'm guessing this `break` is here to avoid compiler warnings, but if we need to add an extra line anyways, I think this is clearer as `abort()` or `exit()` just to show that we don't want it to continue past this point. (And the same after the `execvpe()`.)

Reply:
Agreed; done